proxy/coco/src/request.rs

```diff
@@ -66,7 +66,7 @@ pub struct Request<S, T> {
     attempts: Attempts,
     /// The timestamp of the latest action to be taken on this request.
     #[serde(with = "serde_millis", bound = "T: serde_millis::Milliseconds")]
-    timestamp: T,
+    pub timestamp: T,
```

Comment:
I'd prefer that we have a `fn timestamp(&self) -> &T`. The reason being that `Request` should generally be a blackbox that we can only peek into.